docs/recipes/reducers/04-UsingCombineReducers.md

```diff
@@ -0,0 +1,111 @@
+# Using `combineReducers`
+
+## Core Concepts
+
+
+The most common state shape for a Redux app is a plain Javascript object containing "slices" of domain-specific data at each top-level key.  Similarly, the most common approach to writing reducer logic for that state shape is to have "slice reducer" functions, each with the same `(state, action)` signature, and each responsible for managing all updates to that specific slice of state.  Multiple slice reducers can respond to the same action, independently update their own slice as needed, and the updated slices are combined into the new state object.
+
+Because this pattern is so common, Redux provides the `combineReducers` utility to implement that behavior.  It is an example of a _higher-order reducer_, which takes an object full of slice reducer functions, and returns a new reducer function.
+
+There are several important ideas to be aware of when using `combineReducers`:
+
+- First and foremost, `combineReducers` is simply **a utility function to simplify the most common use case when writing Redux reducers**.  You are *not* required to use it in your own application, and it does *not* handle every possible scenario.  It is entirely possible to write reducer logic without using it, and it is quite common to need to write custom reducer logic for cases that `combineReducer` does not handle.  (See [Beyond `combineReducers`](./05-BeyondCombineReducers.md) for examples and suggestions.)  
+- While Redux itself is not opinionated about how your state is organized, `combineReducers` enforces several rules to help users avoid common errors.  (See [`combineReducers`](../../api/combineReducers.md) for details.)
+- One frequently asked question is whether Redux "calls all reducers" when dispatching an action.  Since there really is only one root reducer function, the default answer is "no, it does not".  However, `combineReducers` has specific behavior that _does_ work that way.  In order to assemble the new state tree, `combineReducers` will call each slice reducer with its current slice of state and the current action, giving the slice reducer a chance to respond and update its slice of state if needed.  So, in that sense, using `combineReducers` _does_ "call all reducers", or at least all of the slice reducers it is wrapping.
+- You can use it at all levels of your reducer structure, not just to create the root reducer.  It's very common to have multiple combined reducers in various places, which are composed together to create the root reducer.
+
+
+## Defining State Shape
```

Comment:
I think that most of the content from this point down is repeated again in the other file, and I think that's a good place for it.  Starting at "`combineReducers` takes an object full of slice reducer functions...", there are parts that probably belong here.  I think I'd defer any discussion of preloadedState to the other document, but anything specific to `combineReducers` can stay here.